mypyc/ir/func_ir.py

```diff
@@ -11,6 +12,7 @@
 )
 from mypyc.ir.rtypes import RType, deserialize_type
 from mypyc.namegen import NameGenerator
+from mypyc.analysis.const_int import find_constant_integer_registers
```

Comment:
I'd prefer if `mypyc.ir` didn't depend on `mypyc.analysis`, to keep import structures less tangled. I think that we can move `mypyc.analysis.const_int` to `mypyc.ir.const_int`, since the analysis it performs is actually quite trivial. If we implement a fancier constant analysis, we'd add it to `mypyc.analysis`, but we wouldn't use it for IR pretty printing.